magefiles/magefile.go

```diff
@@ -41,12 +41,12 @@ func init() {
 }
 
 func version() (string, error) {
-	if ver, err := sh.Output("git", "describe", "--tags", "--always"); err != nil {
+	ver, err := sh.Output("git", "describe", "--tags", "--always")
+	if err != nil {
```

Comment:
I actually prefer inlining if conditions to reduce nesting and lines of code, however this is subjective. But as @knqyf263 pointed out, (not this particular instance) it may lead to subtle errors. And as you mentioned, to suppress them either we'd have to add `preserveScope` argument or `/nolint`.

Based on that is it still worth to add this rule?

Reply:
I prefer not to reuse the variable name inside if-else (for me it is more readable code)
e.g.:
```Go
pkg := Package{name: "foo"}

if p, ok := pkgs["bar"]; ok{ // for this case i don't use `pkg`. I will use `p`, `foundPkg`, etc.
...
}
```

so in my case these changes will not lead to consequences.
but this is just my opinion